components/LocationComparisonBarChart/LocationComparisonBarChart.tsx

```diff
@@ -23,18 +25,45 @@ const LocationComparisonBarChart: FunctionComponent<LocationComparisonChartProps
       trigger: 'axis',
       axisPointer: {
         type: 'shadow'
+      },
+
+      formatter: (params: EChartOption.Tooltip.Format[]): string => {
+        const { seriesName, name, seriesIndex } = params[0];
+        const { value } = params[0] as { value: number };
+
+        if (seriesIndex === 1) {
+          return `<div>${name} <ul><li>${seriesName}: ${addPrefixAndSuffix(
+            formatNumber(value, 1),
+            valueOptions[1]
+          )}</li></ul></div>`;
+        }
+
+        return `<div>${name} <ul><li>${seriesName}: ${addPrefixAndSuffix(
+          formatNumber(value, 1),
+          valueOptions[0]
+        )}</li></ul></div>`;
       }
     },
     xAxis: [
       {
         type: 'value',
-        position: 'top'
+        position: 'top',
+        axisLabel: {
+          formatter: (value: number): string => {
+            return addPrefixAndSuffix(formatNumber(value, 0), valueOptions[0]);
```

Comment:
Could you please check for if the value is 0 and exclude that from the formatting... That way, you'll not have the strange appearance below.

![image](https://user-images.githubusercontent.com/5672438/78545926-479f1900-7805-11ea-8fd8-f02f79a04949.png)